src/garage/tf/misc/tensor_utils.py

```diff
@@ -62,6 +63,25 @@ def filter_valids(t, valid, name='filter_valids'):
     return tf.dynamic_partition(
         t, tf.cast(tf.round(valid), tf.int32), 2, name=name)[1]
 
+def normalize_pixel_observation(env_spec, obs):
```

Comment:
there's a common convention that functions which *might* do something should be called `maybe_foo` rather than just foo.

with that said, see my comments below on why this function shouldn't be checking for input types anyway